main/attitude.h

```diff
@@ -50,14 +50,16 @@ Directional getRealGyro(Directional gyro, Directional offsets){
   return realGyro;
 }
 
-double setGroundLevel() { //Reads bmp data and sets pre-launch altitude
-  double groundLvl;
+void setGroundLevel() { //Reads bmp data and sets ground lvl pressure
   double pressurePreFlight;
-  bmp.performReading();
-  pressurePreFlight = bmp.pressure / 100.0;
-  groundLvl = bmp.readAltitude(pressurePreFlight);
-  return groundLvl;
+  double pressureSum = 0;  //Sum of 100 pressure readings
+  for(int i = 0; i<101; i++){
+    bmp.performReading();
+    pressurePreFlight = bmp.pressure / 100.0; //sample pressure in hPa
+    pressureSum += pressurePreFlight;
   }
+  SEALEVELPRESSURE_HPA = pressureSum / 100.0; //sets average ground lvl pressure
```

Comment:
since we are now overwriting sea level pressure, we can't use `define` at the top of the file. Proper syntax now would be `double SEALEVELPRESSURE_HPA;` at that top line.

Reply:
Also since we are no longer using sea level it should be called ground level